packages/office-ui-fabric-react/src/components/TextField/TextField.tsx

```diff
@@ -131,7 +131,6 @@ export class TextField extends BaseComponent<ITextFieldProps, ITextFieldState> i
       disabled,
       iconClass,
       iconProps,
-      label,
```

Comment:
Is it totally necessary to remove this? I am currently working on implementing required TextField both with a label and without (ie with only a placeholder) and am using this variable as a conditional check.

Reply:
@lynamemi label is always accessible via this.props.label, it was just removed from the deconstruction here because it was pulled out from within the render function below.